srcpkgs/startup/template

```diff
@@ -0,0 +1,66 @@
+# Template file for 'startup'
+pkgname=startup
+version=1.13.3
+revision=1
+build_style=gnu-configure
+configure_args="--disable-sysvcompat --disable-dconf-bridge --disable-cgroups"
+hostmakedepends="autoconf automake libtool pkg-config gettext-devel xz nih-devel"
```

Comment:
xz is part of base-devel

Reply:
I feel like you have told me this before :)